main.go

```diff
@@ -774,10 +893,18 @@ func main() {
 			Action:      connectAction,
 		},
 		{
-			Name:        "disconnect",
+			Name: "disconnect",
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:    "format",
+					Usage:   "prints output of disconnection in machine-readable format (supported formats: \"json\")",
+					Aliases: []string{"f"},
+				},
+			},
 			Usage:       "Disconnects the system from " + Provider,
 			UsageText:   fmt.Sprintf("%v disconnect", app.Name),
 			Description: fmt.Sprintf("The disconnect command disconnects the system from Red Hat Subscription Management, Red Hat Insights and %v and deactivates the %v service. %v will no longer be able to interact with the system.", Provider, ServiceName, Provider),
+			Before:      beforeDisconnectAction,
```

Comment:
Interesting, maybe it could help us to combine "global" and "sub-command" cli options `--format` for v2.

I already did some experiments with combining "global" and "sub-command" cli options with v2, but no workaround work as expected. We can try this in another PR.